uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1652,6 +1653,81 @@ sub computeSNR {
     }
 }
 
+################################################################
+#######################  computeDeepQC #############################
+################################################################
+sub computeDeepQC {
+    my $this = shift;
+    my ($row, $filename, $fileID, $base, $fullpath, $message);
+    my ($tarchiveID, $tarchive_srcloc, $profile)= @_;
+    my $data_dir = NeuroDB::DBI::getConfigSetting(
+                        $this->{dbhr},'dataDirBasepath'
+                        );
+    my $upload_id = getUploadIDUsingTarchiveSrcLoc($tarchive_srcloc);
+    my $ua = LWP::UserAgent->new;
+    my $json = 'JSON::PP'->new;
+    # note: url must be changed to production version later
+    # note 2: include DeepQC modality in prod file
+    my $acqIDForT1Scan = 44;
+    my $query = "SELECT FileID, file, AcquisitionProtocolID from files f ";
+    my $where = "WHERE f.TarchiveSource=?";
```

Comment:
why not simply write the complete query directly here instead of creating it and concatenating?